### PR TITLE
Adding new intro chap for Arm architecture (BSC#1154813)

### PR DESCRIPTION
**Pull request by asettle**
SES has supported ARM architecture since SES 4. Going forward
we want to ensure this is documented for customers. There is
no major difference between the two architectures (x86 and ARM)
but it is important to have documented.

### CODE REVIEW

File: xml/deployment_hwrecommend.xml [hunk 1]
```diff
@@ -23,6 +23,21 @@
...
+    &productname; supports both &x86; and &amd64; architectures.
+    When considering each architecture, it is important to note that
+    from a cores per OSD, frequency, and RAM perspective, there is no real
+    difference between CPU architectures for sizing.
+  </para>
+  <para>
+    However, it is worth noting that erasure code performance on a lower
+    powered &amd64; CPU could be less performant than on an
+    Intel Xeon or higher-end &amd64; CPU.</para>
+</sect1>
```
Comment (dmbyte) [inline xml/deployment_hwrecommend.xml]: NOTE: Like with smaller X86_64 processors (non-server), lower-performance ARM cores may not provide an optimal experience, especially when used for erasure coded pools.

**Comment by asettle** [comment]
Thanks @dmbyte - updated now :)

File: xml/deployment_hwrecommend.xml [hunk 2]
```diff
@@ -23,6 +23,21 @@
   basis. If several processes are located on the same machine, the CPU, RAM,
   disk and network requirements need to be added up.
  </para>
+
+ <sect1 xml:id="multi-architecture">
+  <title>Multiple Architecture Confiurations</title>
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
  <title>Multiple Architecture Configurations</title>
```

File: xml/deployment_hwrecommend.xml [hunk 3]
```diff
@@ -23,6 +23,21 @@
...
+  <para>
+    &productname; supports both &x86; and Arm architectures.
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]: Introduce new entity for 'Arm' please, then use it inside the text.

File: xml/deployment_hwrecommend.xml [hunk 4]
```diff
@@ -23,6 +23,21 @@
...
+    When considering each architecture, it is important to note that
+    from a cores per OSD, frequency, and RAM perspective, there is no real
+    difference between CPU architectures for sizing.
+  </para>
+  <para>
+    Like with smaller &x86; processors (non-server), lower-performance
+    Arm-based cores may not provide an optimal experience, especially when
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]: entity (after you introduce it)

**Comment by asettle** [comment]
@dmbyte @lproven @tbazant ready for reviews :)

File: xml/entity-decl.ent [hunk 5]
```diff
@@ -276,6 +276,8 @@
 
 <!ENTITY x86           "x86">
 <!ENTITY amd64         "AMD64">
+<!ENTITY arm           "Arm">
+<!ENTITY arm64         "&arm;&nbsp;&aarch64;">
```
Comment (tbazant) [inline xml/entity-decl.ent]: i know everything validates from the DocBook perspective, but where do we have the &aarch64; entity defined?

**Pull request merged**